modules/claims_api/spec/requests/v2/veterans/power_of_attorney/2122a_spec.rb

```diff
@@ -5,7 +5,7 @@
 require 'token_validation/v2/client'
 require 'bgs_service/local_bgs'
 
-RSpec.describe 'ClaimsApi::V1::PowerOfAttorney::2122a', type: :request do
+RSpec.describe 'ClaimsApi::V2::PowerOfAttorney::2122a', type: :request do
```

Comment:
This is an unrelated clean up.  This was a typo left over from the big test suite refactor.